xml/x509_subjects.xml

```diff
@@ -1755,4 +1755,20 @@
     <param pos="0" name="os.product" value="Proxmox"/>
   </fingerprint>
 
+  <fingerprint pattern="^CN=(\S{1,512}),OU=Endpoint Health,O=Duo Security\\, Inc.,L=Ann Arbor,ST=Michigan,C=US(?:,\S+)?$">
+    <description>Duo Device Health</description>
+    <example host.name="127.0.0.1">CN=127.0.0.1,OU=Endpoint Health,O=Duo Security\, Inc.,L=Ann Arbor,ST=Michigan,C=US,1.2.840.113549.1.9.1=#0c1e656e64706f696e746865616c74684064756f73656375726974792e636f6d</example>
+    <param pos="0" name="service.vendor" value="Duo"/>
+    <param pos="0" name="service.product" value="Duo Device Health"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+
+  <fingerprint pattern="^CN=(\S{1,512}),OU=Mac Certifier,O=Duo Security\\, Inc.,L=Ann Arbor,ST=Michigan,C=US(?:,\S+)?$">
+    <description>Duo Mac Certifier</description>
+    <example host.name="localhost">CN=localhost,OU=Mac Certifier,O=Duo Security\, Inc.,L=Ann Arbor,ST=Michigan,C=US,1.2.840.113549.1.9.1=#0c18656e64706f696e744064756f73656375726974792e636f6d</example>
+    <param pos="0" name="service.vendor" value="Duo"/>
+    <param pos="0" name="service.product" value="Duo Mac Certifier"/>
```

Comment:
On DUO's website it looks like the product name is `Duo Certifier` ... I'm assuming `Mac` means it's for macOS systems? In which case can we also infer hardware?  Just throwing this out there.

Reply:
You raise an interesting point. I'm going to try to locate a Windows version before merging.

Reply:
The Windows Duo Device Health doesn't appear to be listening on any ports like the macOS software. I updated the fingerprint with `os.*` params.